File: Assignment40/program5.c
//////////////////////////////////////////////////////////////////////
// File name :       program5.c
// Discription :     It is Recusive program to print certain pattern.
// Author :          Mangesh Ashok Bedre
// Date :            1/09/2025
//
//////////////////////////////////////////////////////////////////////

#include<stdio.h>

//////////////////////////////////////////////////////////////////////
// Function name :   Display
// Discription :     It is used to print pattern.
// Input  :          Nothing
// Output :          a  b   c   d   e   f
// Author :          Mangesh Ashok Bedre
// Date :            1/09/2025
//
//////////////////////////////////////////////////////////////////////

void Display()
{
    static char ch = 'a';
    
    if(ch > 'f')
    {
        return;

    }
    else
    {
        printf("%c\t",ch);
        ch++;
        Display();
    }
}

int main()
{
    Display();
    return 0;
}

File: Assignment46/program5.java

import java.util.Scanner;
//////////////////////////////////////////////////////////////////////
// File name :       program5.java
// Discription :     It is a program to print  pattern.
// Author :          Mangesh Ashok Bedre
// Date :            25/11/2025
//
//////////////////////////////////////////////////////////////////////
class Pattern
{
    
    //////////////////////////////////////////////////////////////////////
    // Function Name :   Display
    // Discription :     It is used to print pattern.
    // Input  :          int ,int
    // Output :          1 2 3 4
    //                   5 6 7 8
    //                   9 10 11 12
    // Author :          Mangesh Ashok Bedre
    // Date :            25/11/2025
    //
    //////////////////////////////////////////////////////////////////////
    public void Display(int rows, int cols)
    {
        int i = 0;
        int j = 0,iNo  = 0;

        for(i = rows,iNo = 1 ; i >= 1; i--)
        {
            for(j = 0; j < cols; j++,iNo++)
            {
                System.out.print(iNo+"\t");
            }
            System.out.println();
        }
    }
}

class program5
{
    public static void main(String a[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the rows : ");
        int rows = sobj.nextInt();

        System.out.println("Enter the cols : ");
        int cols = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(rows,cols);
    }
}